CASTEPbands/spgutils.py

```diff
@@ -29,12 +29,6 @@ def _get_bravais_lattice_spg(cell):
       saving on some duplication of code.
 
     """
-    # Get all the Bravais lattices
-    bv_dict = latt.bravais_lattices
-
-    # Get the cell's lattice parameters
-    a, b, c, alpha, beta, gamma = cell.cell.cellpar()
```

Comment:
There are no longer used in this routine and thus they just take up the namespace.